lib/Controller/MetaDataController.php

```diff
@@ -115,21 +122,23 @@ public function setMetaData(int $id, string $metaData): DataResponse {
 	/**
 	 * Update metadata
 	 *
+	 * @PublicPage
```

Comment:
Important: the `PUT /.../meta-data/{id}` is now open to public access

Reply:
I think this is not good, as then everyone can overwrite the current metadata.
This should be a separate endpoint, which only allows to append filedrop information.
(I discussed it back then with Carl this way…)